Miners/CcminerTpruvot.ps1

```diff
@@ -1,7 +1,7 @@
 ﻿using module ..\Include.psm1
 
 $Path = ".\Bin\NVIDIA-TPruvot\ccminer-x64.exe"
-$Uri = "https://github.com/tpruvot/ccminer/releases/download/2.2.4-tpruvot/ccminer-x64-2.2.4-cuda9.7z"
+$Uri = "https://github.com/MSFTserver/ccminer/releases/download/2.2.5-rvn/ccminer-x64-2.2.5-rvn-cuda9.7z"
```

Comment:
Can this still be called CcminerTpruvot when it's coming from a different source?

Reply:
Yes, It is by the same author and just another version of the same miner
Don't know why Tpruvot uses a different repo.